documentation/jetty-documentation/pom.xml

```diff
@@ -251,6 +266,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>attach-javadoc</id>
+            <phase>package</phase>
+            <goals>
+              <goal>attach-artifact</goal>
+            </goals>
+            <configuration>
+              <artifacts>
+                <artifact>
+                  <file>${project.build.directory}/other-artifacts/javadoc-${project.version}-javadoc.jar</file>
```

Comment:
maybe as it's documentation (the other one is .zip) and a full aggregated Javadoc I would expect more of a .zip file (but really personal opinion :) )

Reply:
We haven't been able to use the aggregate javadoc build since Jetty 10 and our move to JPMS.

The new maven-javadoc-plugin (3.6.0) has a new `<legacyMode>true</legacyMode>` configuration that has promise (for us).

I'm experimenting with it now (in this branch / PR).

Just fighting the maven-dependency-plugin rules ATM, had to make a 3.6.1-SNAPSHOT build locally so I can figure out what I need to change.
See https://github.com/joakime/maven-dependency-plugin/commit/52631866d9f9a2f53bebf11ac44270aea0a7fc11